### PR TITLE
service status: added references to service config pages

### PR DESCRIPTION
**Pull request by 8191**
- Added links to main pages, log pages, and status pages of services shown in the System: Diagnosics: Services page.

- Changed incorrect name of unbound service from "Unbound DNS Forwarder" to "Unbound DNS Resolver"

### CODE REVIEW

**Comment by 8191** [comment]
I've changed a `gettext` obtained string, shall I adopt the language file, or will this be done before release of a new version?

**Comment by fichtner** [comment]
I'm working to get rid of shortcuts.inc. It's trying to do too many things at once and is inconsistent while doing so (some pages navigate to a log page, but the log page is for two facilities so it routes back the wrong way...) and the service start/stop widget ties into this as well.

For now, all items except 'service' should be zapped pending more menu consolidation. It shouldn't be necessary.

For gettext(): if it's necessary it's just changed, the full regeneration is only executed every few weeks to keep the lines changed churn minimal, see this workflow:

https://github.com/opnsense/core/commit/f2e052754f4539d7b066e73ac5a4e13ab84cb153
https://github.com/opnsense/core/commit/08749b414f0c2c5a67737383296abfe503d7dc8f
https://github.com/opnsense/core/commit/1d744002922155c841ab043650be4525cd5f206d

Translation progress from Pootle is mainly merged before an update:

https://github.com/opnsense/core/commit/42c658c73f8c92fa573188a7078ff4666da1267d

Must-see ticket for the DNS resolver/forwarder controversy: https://github.com/opnsense/core/issues/354

**Comment by 8191** [comment]
Agreed, the shortcuts.inc is something awful. Nevertheless currently it's a bit awkward finding all the different sections for each service. What about adding these icons until a better way of navigation comes up?

**Comment by fichtner** [comment]
I've kept logs and service entries for now, but killed status and main. If they are not correctly grouped in the menu they should be reordered to be correctly grouped there (Menu.xml).

File: src/www/shortcuts.inc [hunk 1]
```diff
@@ -190,3 +200,11 @@ $shortcuts['squid'] = array();
...
 $shortcuts['squid']['log'] = "diag_logs_proxy.php";
 $shortcuts['squid']['service'] = "squid";
+
+$shortcuts['sshd'] = array();
+$shortcuts['sshd']['main'] = 'system_advanced_admin.php';
+$shortcuts['sshd']['service'] = 'sshd';
+
+$shortcuts['radvd'] = array();
+$shortcuts['radvd']['main'] = 'services_dhcpv6.php';
+$shortcuts['radvd']['service'] = 'radvd';
```
Comment (fichtner) [inline src/www/shortcuts.inc]: are those shortcuts even used? see $shortcut_section assigment on individual pages

**Comment by fichtner** [comment]
Ok, here's what we can do: keep/add log entries where the log entries for shortcuts that are not in the same submenu (e.g. we don't need a squid log shortcut). Fix all currently available shortcuts to appear on pages that are unambiguous and not a log page (e.g. like upnp on the settings and status page)

**Comment by fichtner** [comment]
Thanks, pushed the rename part manually, no more shortcut links though. See https://github.com/opnsense/core/commit/832b801195165b7f2cd31f0186f382edc631a1f4

**Pull request closed without merging**